plugin/src/main/java/io/jenkins/plugins/forensics/git/delta/GitDeltaCalculator.java

```diff
@@ -60,4 +63,17 @@ public GitDeltaCalculator(final GitClient git) {
 
         return Optional.empty();
     }
+
+    @Override
+    public Optional<Delta> calculateDelta(final Run<?, ?> build, final Run<?, ?> referenceBuild,
+            final FilteredLog logger) {
+        String scm = new ScmResolver().getScm(build).getKey();
```

Comment:
I think this will not work in general as pipelines can use multiple SCMs. 

We probably need to add a similar configuration of https://github.com/jenkinsci/warnings-ng-plugin/blob/master/plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java#L181-L195 to the `CoveragePublisher`. Then we can use an additional parameter here in this method.